docs/debugging-reconfigurator.adoc

```diff
@@ -0,0 +1,1466 @@
+
+```
+root@oxz_switch0:~# omdb nexus background-tasks print-report blueprint_executor
+note: Nexus URL not specified.  Will pick one from DNS.
+note: using DNS server for subnet fd00:1122:3344::/48
+note: (if this is not right, use --dns-server to specify an alternate DNS server)
+note: using Nexus URL http://[fd00:1122:3344:10b::3f]:12232
+[Nov 05 23:13:35]   Running ( 1/15) Ensure external networking resources
+[Nov 05 23:13:36] Completed ( 1/15) Ensure external networking resources: after 1.02s
+[Nov 05 23:13:36]   Running ( 2/15) Fetch sled list
+[Nov 05 23:13:36] Completed ( 2/15) Fetch sled list: after 93.33ms
+[Nov 05 23:13:36]   Running ( 3/15) Ensure db_metadata_nexus_state records exist
+[Nov 05 23:13:36] Completed ( 3/15) Ensure db_metadata_nexus_state records exist: after 54.31ms
+[Nov 05 23:13:36]   Running ( 4/15) Deploy sled configs
+[Nov 05 23:13:38] Completed ( 4/15) Deploy sled configs: after 1.92s
+[Nov 05 23:13:38]   Running ( 5/15) Plumb service firewall rules
+[Nov 05 23:13:39] Completed ( 5/15) Plumb service firewall rules: after 810.77ms
+[Nov 05 23:13:39]   Running ( 6/15) Deploy DNS records
+[Nov 05 23:13:39] Completed ( 6/15) Deploy DNS records: after 173.48ms
+[Nov 05 23:13:39]   Running ( 7/15) Cleanup expunged zones
+[Nov 05 23:13:40] Completed ( 7/15) Cleanup expunged zones: after 453.93ms
+[Nov 05 23:13:40]   Running ( 8/15) Decommission sleds
+[Nov 05 23:13:40] Completed ( 8/15) Decommission sleds: after 243.18ms
+[Nov 05 23:13:40]   Running ( 9/15) Decommission expunged disks
+[Nov 05 23:13:41] Completed ( 9/15) Decommission expunged disks: after 760.83ms
+[Nov 05 23:13:41]   Running (10/15) Deploy clickhouse cluster nodes
+[Nov 05 23:13:41] Completed (10/15) Deploy clickhouse cluster nodes: after 427.06ms
+[Nov 05 23:13:41]   Running (11/15) Deploy single-node clickhouse cluster
+[Nov 05 23:13:41] Completed (11/15) Deploy single-node clickhouse cluster: after 119.64ms
+[Nov 05 23:13:41]   Running (12/15) Mark support bundles as failed if they rely on an expunged disk or sled
+[Nov 05 23:13:41] Completed (12/15) Mark support bundles as failed if they rely on an expunged disk or sled: after 65.81ms with message: support bundle expunge report: SupportBundleExpungementReport { bundles_failed_missing_datasets: 0, bundles_deleted_missing_datasets: 0, bundles_failing_missing_nexus: 0, bundles_reassigned: 0 }
+[Nov 05 23:13:41]   Running (13/15) Reassign sagas
+[Nov 05 23:13:42] Completed (13/15) Reassign sagas: after 155.91ms
+[Nov 05 23:13:42]   Running (14/15) Ensure CockroachDB settings
+[Nov 05 23:13:42] Completed (14/15) Ensure CockroachDB settings: after 16.33ms
+[Nov 05 23:13:42]   Running (15/15) Kick off MGS-managed updates
+[Nov 05 23:13:42] Completed (15/15) Kick off MGS-managed updates: after 6.43µs
+```
+
+Problems during blueprint execution may show up in the `error` field above or in the report.
```

Comment:
Nit - most problems show up under `warning`, not `error`. (We reserved `error` for the small number of steps that prevent further execution - mostly "I couldn't talk to CRDB early in the attempt").